### PR TITLE
Ability to set environmental variables

### PR DESCRIPTION
**Pull request by Gytiss**
References #14 
This pull request allows to pass environmental variables via key _environment_  in json configuration file
This makes key _environment_ mandatory, eg:
```json
{
  "dockerRestApiUri": "http://localhost:2375",
  "localhost": "0.0.0.0",
  "dockerImagePort": "4444",
  "maxSession": 10,
  "mapping": [
    {
      "browser": "chrome",
      "target": "selenium/standalone-chrome:latest",
      "implementation": "com.rationaleemotions.server.DockerBasedSeleniumServer"
    },
    {
      "browser": "firefox",
      "target": "selenium/standalone-firefox-debug:latest",
      "implementation": "com.rationaleemotions.server.DockerBasedSeleniumServer"
    }
  ],
  "environment": {
     "SCREEN_WIDTH": 768,
     "SCREEN_HEIGHT": 24
  }
}
```
Or it can be left empty eg:
```json
"environment": {
}
```
If approved, the readme should be updated as well with new json key

Also changed session counter to start from 0

### CODE REVIEW

**Comment by krmahadevan** [comment]
@Gytiss - Can you also include the `README.md` changes as well in this PR ?

File: src/main/java/com/rationaleemotions/proxy/GhostProxy.java [hunk 1]
```diff
@@ -32,7 +32,7 @@
  * and then routing the session traffic to the spawned server.
  */
 public class GhostProxy extends DefaultRemoteProxy {
-    private final AtomicInteger counter = new AtomicInteger(1);
+    private final AtomicInteger counter = new AtomicInteger(0);
```
Comment (krmahadevan) [inline src/main/java/com/rationaleemotions/proxy/GhostProxy.java:35]: Any specific reason why this was changed ?

**Comment by Gytiss** [comment]
> @Gytiss - Can you also include the `README.md` changes as well in this PR ?

Changes included

**Review by krmahadevan — COMMENTED** [review]
@Gytiss - Thanks for this PR. Can you please let me know why we are updating the `AtomicInteger counter` from `0` to `1` ?

**Comment by Gytiss** [comment]
> @Gytiss - Thanks for this PR. Can you please let me know why we are updating the `AtomicInteger counter` from `0` to `1` ?

I actually update it from 1 to 0. The reason is that if I start only one session, the counter will show 2 in console, thus suggesting that you have two containers up. Also in SpawnedServer class I saw this:
`AtomicInteger attempts = new AtomicInteger(0);`, which is kind of inconsistant, so I though I'll make it the same in GhostProxy class.

**Review by krmahadevan — APPROVED** [review]

**Comment by krmahadevan** [comment]
@Gytiss - Thanks for the PR. This stands merged

**Pull request merged**